### DISCUSSION TITLE
v0.2.0-alpha

### DISCUSSION THREAD

**Discussion by Jazcash** [beyond-all-reason/bar-lobby#5] [Show and tell]
<hr /><em>This discussion was created from the release <a href='https://github.com/Jazcash/bar-lobby/releases/tag/v0.2.0-alpha'>v0.2.0-alpha </a>.</em>

**Comment by icexuick** [comment]
Shaping up quickly and very nicely.
For now i have nothing to nitpick about! Looking fantastic!

**Reply by icexuick** [reply]
Maybe 1 thing: The animation when changing screens is a bit too flashy and is a bit nauseating for me. Maybe swoosh to left/right, or try something more overlapping, so the main BG does not flash up sharp/bright between pages.
